src/components/UserProfileCard.tsx

```diff
@@ -0,0 +1,25 @@
+import { Grid, Modal } from "@material-ui/core"
+import styled from "styled-components"
+import { User } from "../services/api/types/User"
+
+const ProfileTitle = styled(Grid)`
+  text-align: center;
+`
+
+interface profileProps {
```

Comment:
```suggestion
interface ProfileProps {
```
type nên kiểu PascalCase, camelCase dành cho biến và hàm